### PR TITLE
add conversation design get started

### PR DESCRIPTION
**Pull request by Zeng666666**
#1196  merged，再 approve 这个。因为我把 fix revert 了。

### CODE REVIEW

File: docs/reference/conversation-design/get-started.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# How do I get started? 
+
+Conversation interaction is different from GUI interaction in that users are not restricted to following a linear path. They may jump around, express themselves freely, and take unexpected turns. This means that when designing a conversation, you need to think about user interaction in a non-linear way.
+
+You cannot simply map out a linear path and expect users to follow it. Instead, you need to consider the core of your business, the experience you want to bring to users, and how users can get services through interaction. The key is to think about the specific goals of your conversation and to design it accordingly. 
+
+## Gathering requirements
+First, identify the core of your business. What are the essential services that you offer? What kind of information do your users need to provide? What are the key features that you want to highlight? Gathering requirements is the process of understanding the needs of your customers and stakeholders. It is essential for any business that wants to develop a successful product or service. This is all part of **gathering requirements**.
+
+## Design interaction
```
Comment (xiaoyunwu) [inline docs/reference/conversation-design/get-started.md:10]: this is should be moved to after the use case.

File: docs/reference/conversation-design/get-started.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Testing and iterating
+As soon as you have some sample use cases, it is important to start **testing and iterating** on your designs. This is because it can be difficult to identify problems when you are immersed in the design process. An outsider's opinion can be helpful in identifying problems that you may not have noticed. The good news is that you can get insight into whether your design will work for users before writing a single line of code.
+
+## Design for the long tail
+In the last phase of the process, you will specify the detailed design for your conversational feature. This involves making sure your feature adequately covers the **long tail** of ways a conversation can deviate from the most common paths. To do this, you will need to add handling for errors and other uncommon scenarios.
+
+## Scale your design
```
Comment (xiaoyunwu) [inline docs/reference/conversation-design/get-started.md:22]: what do you mean by scale?
more skills? more channels? more expressions?

File: docs/reference/conversation-design/get-started.md (hunk 1 above)
Comment (Zeng666666) [reply docs/reference/conversation-design/get-started.md:10]: 指的哪个？头三个其实是我们 CUI design，先确定 business 和 schema level，再确定 interaction logic，再写 use case

File: docs/reference/conversation-design/get-started.md (hunk 2 above)
Comment (Zeng666666) [reply docs/reference/conversation-design/get-started.md:22]: more channels，more expressions，more replies

**Review by xiaoyunwu — APPROVED** [review]

**Pull request merged**